libnetwork/store_test.go

```diff
@@ -76,6 +76,6 @@ func OptionBoltdbWithRandomDBFile(t *testing.T) config.Option {
 
 	return func(c *config.Config) {
 		c.DatastoreConfig.Path = tmp
-		c.DatastoreConfig.Config = store.Config{Bucket: "testBackend"}
+		c.DatastoreConfig.Config = boltdb.Config{Bucket: "testBackend"}
```

Comment:
Hm... ok; perhaps we'd still need some way to set it, but it look like _ONLY_ in tests? And _ONLY_ internal to the package, so .. perhaps we can wire that up in tests only.